src/Components/Alert/Components/Modal/ModalContent.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react"
+import { useSentinelVisibility } from "@artsy/palette/dist/utils/useSentinelVisibility"
+import { DROP_SHADOW } from "@artsy/palette/dist/helpers/shadow"
+import { Box, BoxProps } from "@artsy/palette/dist/elements/Box/Box"
+import { Flex, ModalClose } from "@artsy/palette"
+
+export interface ModalDialogContentProps
+  extends BoxProps,
+    React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode
```

Comment:
Shouldn't need to type `children`, as that comes with `React.FC`

Reply:
Good catch, removed it!